modules/runner/src/integrationTest/java/org/apache/ignite/internal/sqllogic/ItSqlLogicTest.java

```diff
@@ -137,11 +135,10 @@
  * @see <a href="https://www.sqlite.org/sqllogictest/doc/trunk/about.wiki">Extended format documentation.</a>
  */
 @Tag(value = "sqllogic")
-@ExtendWith({WorkDirectoryExtension.class, SystemPropertiesExtension.class})
```

Comment:
Looks like `SystemPropertiesExtension` is still needed

Reply:
It is registered by this classes' grandfather

Reply:
After thinking a bit, I specified the extension explicitly